source/crud/crud.md

```diff
@@ -2199,6 +2199,23 @@ the [$readPreference global command argument](../message/OP_MSG.md#global-comman
 [passing read preference to mongos and load balancers](../server-selection/server-selection.md#passing-read-preference-to-mongos-and-load-balancers)
 (if applicable).
 
+### Explain
+
+Drivers MAY provide explain helpers. If a driver does provide explain helpers, the driver MUST ensure that its helper
+permits users to specify maxTimeMS for the explain command specifically. An example, using Node, might look like:
```

Comment:
Based on the discussion in the ticket, we should loosen the language here. My suggestion is to say something like "permits users to specify a timeout using the maxTimeMS or timeoutMS option ..."